lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
Actually 'data' is an array, using 'sendChunk' unmodified will result in something like that:
{ data: [ { correlationIds: [1,2] } ], ... }
I can create a 'sendCorrelationIds' function which adds a top property to the response, that would be better than tweaking either 'sendChunk' or 'sendEnd'.